config/locales/es.yml

```diff
@@ -2047,7 +2047,7 @@ es:
             <li>Una declaración federal de 2023 <strong>aceptada</strong> mediante <a href="https://directfile.irs.gov/">IRS Direct File</a>. (No necesitas una copia de esta declaración; transferirás tu declaración aquí).</li>
             <li>Dirección de correo electrónico o número de teléfono.</li>
             <li>Licencia de conducir o identificación emitida por el estado.</li>
-            <li>Formulario 1099-G (si recibiste beneficios de desempleo en 2023).</li>
+            <li class="hide--nj">Formulario 1099-G (si recibiste beneficios de desempleo en 2023).</li>
```

Comment:
I had some trouble formulating my thoughts arounds this. On one hand it does seem to solve the issue concisely and work well which I like but on the other hand its a departure from what we usually do in putting this kind of logic in the view and might obscure it a bit or get lost in transifex. [boulder] I think we definitely should add some sort of test or explore doing this in some other way like putting it into the view. What do you think?

Reply:
Totally agree. I took this approach because I have been trying to use only the lightest touch possible in any files that aren't NJ specific. Is it possible instead to have default text and override if there's state specific text?

Reply:
Yes! Do mean adding a new jersey specific `help_text_html` key in the yml and then saving the original `help_text_html` as the default and displaying them in some sort of:
```
<% if current_state_code == 'nj' %>
<%= t("nj.help_text_html") %>
<% else %>
<%= t(".help_text_html") %>
<% end %>
```
I think that would work! Sucks to have to add the whole block of text again but does help with keeping the logic in one place